### PR TITLE
FIP-40 typescript SDK changes to add permissions getters

### PR DESCRIPTION
**Pull request by edrotthoff**
fiosdk.html changes are in progress, please examine the code here and comment.

FIP-40 getter integration into fio sdk, 
BD-4559 get_grantee_permissions, 
BD-4560 get_grantor_permissions, 
BD-4561 get_object_permissions

### CODE REVIEW

File: src/FIOSDK.ts [hunk 1]
```diff
@@ -1593,6 +1631,13 @@ export class FIOSDK {
         return this.getFeeForTransferLockedTokens(params.fioAddress)
       case 'getObtData':
         return this.getObtData(params.limit, params.offset, params.tokenCode)
+      case 'getGranteePermissions':
+        return this.getGranteePermissions(params.granteeAccount, params.limit, params.offset)
+      case 'getGrantorPermissions':
+        return this.getGrantorPermissions(params.grantorAccount, params.limit, params.offset)
+      case 'getObjectPermissions':
+        return this.getObjectPermissions(params.permissionName, params.objectName, params.limit, params.offset)
+
```
Comment (trukhilio) [inline src/FIOSDK.ts]: Let's remove empty line from here

File: tests/index.spec.js [hunk 2]
```diff
@@ -139,6 +145,67 @@ before(async () => {
...
+          permission_name: permName,
+          permission_info: "",
+          object_name: newFioDomain,
+          max_fee: defaultFee,
+          tpid: '',
+          actor: fioSdk.account
+        }
+      })
+
+      //console.log(result);
```
Comment (trukhilio) [inline tests/index.spec.js]: Let's remove `console.log`

**Review by trukhilio — APPROVED** [review]
Please, check comments

File: src/FIOSDK.ts (hunk 1 above)
Comment (edrotthoff) [reply src/FIOSDK.ts]: removed

File: tests/index.spec.js (hunk 2 above)
Comment (edrotthoff) [reply tests/index.spec.js]: will do

**Pull request merged**